### PR TITLE
doc: Robotoff doc update

### PR DESCRIPTION
**Pull request by Jagrutiti**
### What
Updating Robotoff document

### Closes
- #689

### CODE REVIEW

File: doc/introduction/contributing.md [hunk 1]
```diff
@@ -73,6 +82,13 @@ Ready to contribute code? Here's how to set up Robotoff for local development.
     poetry run pytest tests
     ```
 
+    Before running the test cases make sure you have a database created. Have a look at [settings.py](../../robotoff/settings.py#L131) the default database name, user, and password is
```
Comment (alexgarel) [inline doc/introduction/contributing.md:85]: I would better say to look at `.env` to get the username and password

**Review by alexgarel — CHANGES_REQUESTED** [review]
Thank you for the PR, good, I just require a small change.

File: doc/introduction/contributing.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ Ready to contribute code? Here's how to set up Robotoff for local development.
     poetry run pytest tests
     ```
 
-    Before running the test cases make sure you have a database created. Have a look at [settings.py](../../robotoff/settings.py#L131) the default database name, user, and password is
+    Before running the test cases make sure you have a database created. Have a look at [.env](../../robotoff/settings.py#L131) the default database name, user, and password is
```
Comment (alexgarel) [inline doc/introduction/contributing.md:85]: I think this explanation is for non docker user, so the .env is not relevant to them.

Comment (Jagrutiti) [reply doc/introduction/contributing.md:85]: @alexgarel then what shall I name it?

Also, I have to add the documentation for https://github.com/openfoodfacts/robotoff/pull/693. Shall I add this documentation in this PR? Or make a new one?

Comment (alexgarel) [reply doc/introduction/contributing.md:85]: the documentation for https://github.com/openfoodfacts/robotoff/pull/693 should be in #693 PR !

Comment (Jagrutiti) [reply doc/introduction/contributing.md:85]: My bad. 
I realized that later. I have added the documentation in #693  PR itself.

Comment (alexgarel) [reply doc/introduction/contributing.md:85]: Hum indeed you put .env but you ling to `settings.py` :-) 

```suggestion
    Before running the test cases make sure you have a database created. Have a look at [robotoff/settings.py](../../robotoff/settings.py#L131) the default database name, user, and password is
```

File: doc/introduction/contributing.md [hunk 3]
```diff
@@ -73,6 +82,13 @@ Ready to contribute code? Here's how to set up Robotoff for local development.
     poetry run pytest tests
     ```
 
+    Before running the test cases make sure you have a database created. Have a look at [.env](../../robotoff/settings.py#L131) the default database name, user, and password is
+
+    ```
+    postgres
+    ```
+    Configure them as you like. We don't provide sample database as of now but you'll have a database structure ready to start working.
```
Comment (alexgarel) [inline doc/introduction/contributing.md:90]:
```suggestion
    Configure them through environment (you may use `.env` if you use docker) as you like. We don't provide sample database as of now but you'll have a database structure ready to start working.
```

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**